Preprocessing/MergedDataFrames.ipynb

```diff
@@ -0,0 +1,1522 @@
+       "0            452600.0        880.0     129.0       126.0        NEAR BAY   \n",
+       "1            358500.0       7099.0    1106.0      1138.0        NEAR BAY   \n",
+       "2            352100.0       1467.0     190.0       177.0        NEAR BAY   \n",
+       "3            341300.0       1274.0     235.0       219.0        NEAR BAY   \n",
+       "4            342200.0       1627.0     280.0       259.0        NEAR BAY   \n",
+       "\n",
+       "   Max Temp  Humidity  Cloudiness  Wind Speed Description   County  Employees  \\\n",
+       "0     88.09      65.0        20.0       10.36  few clouds  Alameda   505595.0   \n",
+       "1     87.76      62.0        20.0       10.36  few clouds  Alameda   505595.0   \n",
+       "2     87.76      62.0        20.0       10.36  few clouds  Alameda   505595.0   \n",
+       "3     88.09      65.0        20.0       10.36  few clouds  Alameda   505595.0   \n",
+       "4     88.09      65.0        20.0       10.36  few clouds  Alameda   505595.0   \n",
+       "\n",
+       "   Establishments  \n",
+       "0         32808.0  \n",
+       "1         32808.0  \n",
+       "2         32808.0  \n",
+       "3         32808.0  \n",
+       "4         32808.0  "
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Final merge.\n",
+    "merged_df = housing_weather_df.merge(population_census_df, on='City', how='left')\n",
+    "\n",
+    "print(merged_df.shape)\n",
+    "merged_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "id": "47fc0b18",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "City                     0\n",
+       "Longitude                0\n",
+       "Lattitude                0\n",
+       "Population               0\n",
+       "Median Age               0\n",
+       "Median Income            0\n",
+       "Median House Value       0\n",
+       "Total Rooms              0\n",
+       "Bedrooms                 0\n",
+       "Households               0\n",
+       "Ocean Proximity          0\n",
+       "Max Temp                 0\n",
+       "Humidity                 0\n",
+       "Cloudiness               0\n",
+       "Wind Speed               0\n",
+       "Description              0\n",
+       "County                8932\n",
+       "Employees             8932\n",
+       "Establishments        8932\n",
+       "dtype: int64"
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Check for null values.\n",
+    "merged_df.isnull().sum()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "id": "1c6e7496",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Drop null values.\n",
+    "merged_df = merged_df.dropna()"
```

Comment:
I think, we should retain the data-set in cell # 40 in a separate variable, so that we can base our model off this dataframe as well. Since this dataframe has almost 20k rows as compared to the final merged_df that has only ~10k rows. We can choose to use the best model of the two in our final submission. We can also establish whether the added columns had any impact on the performance of the model or not in our analysis.

Reply:
I believe I made the changes. I don't know if I need to make a new pull request so let me know if you can see them!